PalasoUIWindowsForms/ImageToolbox/ImageToolboxControl.cs

```diff
@@ -218,11 +214,9 @@ private void listView1_SelectedIndexChanged(object sender, EventArgs e)
 
 				if (_currentControl != null)
 				{
-					GetImageFromCurrentControl();
-
 					_panelForControls.Controls.Remove(_currentControl);
 					((IImageToolboxControl) _currentControl).ImageChanged -= new EventHandler(imageToolboxControl_ImageChanged);
-					_currentControl.Dispose();
```

Comment:
I don't have the big picture entirely. At least in this method, _currentControl comes from running a function which is the tag of the list item. Are these functions something the ImageCropper initializes, or something its client initializes? Either way, it should be documented either that each call of one of these functions creates a new control which the ImageCropper will dispose, or that the client that supplies the function will be responsible to dispose of it. I lean towards the former, since otherwise the client has no way to know when it can dispose the control. I'm not happy with leaving disposable objects to be disposed by their finalizers; in general, there is no way of knowing what important resources might need to be released in a timely way. Also code-checking tools complain about it. But I don't know how feasible this is or how far out into the client community such a change might affect things.